app/components/viewer/TimeLine.tsx

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Perhaps add these inline for the cases where they are really necessary? Just like you did at some other spots